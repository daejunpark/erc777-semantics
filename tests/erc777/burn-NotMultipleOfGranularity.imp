test.preamble();
test.setGranularity(10)

// Testing burn(Value) when Value is not multiple of granularity().

test.burn(23);
