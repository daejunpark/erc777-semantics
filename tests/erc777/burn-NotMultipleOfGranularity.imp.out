granularity: 10
Burning 23 units from 7
REVERT
Balance of 7 is 70
Total supply is 550
